capture-sdk/sdk/src/main/java/net/gini/android/capture/GiniCapture.java

```diff
@@ -171,20 +179,79 @@ public static synchronized Builder newInstance() {
      *
      * @param context Android context
      */
-    public static synchronized void cleanup(@NonNull final Context context) {
-        if (sInstance != null) {
-            sInstance.mDocumentDataMemoryCache.clear();
-            sInstance.mPhotoMemoryCache.clear();
-            if (sInstance.mNetworkRequestsManager != null) {
-                sInstance.mNetworkRequestsManager.cleanup();
-            }
-            if (sInstance.mGiniCaptureNetworkApi != null) {
-                sInstance.mGiniCaptureNetworkApi.setUpdatedCompoundExtractions(Collections.<String, GiniCaptureCompoundExtraction>emptyMap());
-            }
-            sInstance.mImageMultiPageDocumentMemoryStore.clear();
-            sInstance.internal().setReviewScreenAnalysisError(null);
-            sInstance = null; // NOPMD
+    public static synchronized void cleanup(@NonNull final Context context,
```

Comment:
[Request] We also updated this method and we should document what's the purpose for the new params and also list them with `@param`.